pkgs/by-name/i2/i2pd/package.nix

```diff
@@ -53,6 +53,7 @@ stdenv.mkDerivation rec {
     homepage = "https://i2pd.website";
     description = "Minimal I2P router written in C++";
     license = licenses.bsd3;
+    maintainers = with maintainers; [ edwtjo ];
```

Comment:
optional:

Compared to all the other packages, `i2pd` did have quite a few PRs over time, but it was hard for you to react to them, too.

I suggest to drop maintainership of this as well - it will already be hard enough for you to keep up with the notifications going forward.

(and yes - reacting to r-ryantm PRs / auto-updates is *also* expected)